packages/nextjs/components/NetworksDropdown.tsx

```diff
@@ -47,11 +59,23 @@ const groupedOptions = networks.reduce<GroupedOptions>(
   },
 );
 
+groupedOptions.mainnet.options.push({
+  value: "see-all",
+  label: "See All Chains",
+  icon: "EyeIcon",
+});
+
+const allChains = Object.values(chains).map(chain => ({
```

Comment:
Should we filter out the `getPopularTargetNetworks` form this allChains so in modal user sees only chain that are not present in direct dropdown ? 

Or maybe its good idea to show them all only? I think both looks fine

Reply:
I think your suggestion makes a lot of sense. Let's ask @Pabl0cks what he thinks.

Reply:
I think it's a good idea to filter them out, and maybe change the title from "See All Chains" to "Other Chains".

Probably the users that are going to use this feature are the ones that can't find the chain they need on the dropdown, and since dropdown is so clear with icons, Mainnet/Testnet separations and such, probably users won't miss the chain if it's in there.

Maybe is also a good idea to move "See All Chains"/"Other Chains" at the end of the dropdown? Instead of the end of Mainnet section. Specially if we decide to filter popular networks out, so we make sure users have scrolled down through all the chains before searching for a different one.

Reply:
> Maybe is also a good idea to move "See All Chains"/"Other Chains" at the end of the dropdown? I

yeah even I had this thought in mind, +1 for this